en/contribute/language.md

```diff
@@ -0,0 +1,12 @@
+Language
+========
+
+The development language is English. All comments and documentation in common repositories should be written in English, so that contributions can be made by developers around the world.
+
+
+Country-specific repositories
+-----------------------------
+
+Some repositories define the social and fiscal system of a specific country. In such cases, the language used throughout issues and pull requests should be one of the native ones of that country.
+
```

Comment:
An example could have made my understanding more direct, like so:

For instance [OpenFisca-France](https://github.com/openfisca/openfisca-france/) issues and pull-requests should be written in French.